docker-compose.build.yml

```diff
@@ -0,0 +1,71 @@
+version: '3'
+services:
+  # Server
+  server_npm:
+    build:
+      context: 'server/'
+      dockerfile: 'Dockerfile.packages'
+    image: liberoadmin/reviewer_server_packages:${IMAGE_TAG}
+    networks: 
+        - 'reviewer_build'
+  server_typescript:
+    build:
+      context: 'server/'
+      dockerfile: "Dockerfile.typescript"
+      args:
+        image_tag: ${IMAGE_TAG}
+    image: liberoadmin/reviewer_server_typescript:${IMAGE_TAG}
+    depends_on:
+      - server_npm
+    networks: 
+        - 'reviewer_build'
+  reviewer_server:
```

Comment:
why the two services have a prefix (`reviewer_server:`, `reviewer_client`) and the rest does not? Seems unnecessary given `docker-compose` will already use the `reviewer` folder as a prefix for container, network, volume names

Reply:
Because eventually there are going to be other services and the whole thing is going to be renamed then